### PR TITLE
piglow: various readability improvements

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

File: piglow/examples/strobe.go [hunk 1]
```diff
@@ -16,42 +13,24 @@ func main() {
 		panic(err)
 	}
 
-	// catch signals and terminate the app
```
Comment (mattetti) [inline piglow/examples/strobe.go]: I have to admit that it is a better example. It's easier to read, it exits cleanly and doesn't have to use a signal. Note that the signal handling can be documented in the overall README or wiki (I do think it's something real users will want to want to learn about, but I now see that it doesn't have to be in the default examples).

File: piglow/piglow.go [hunk 2]
```diff
@@ -34,29 +30,24 @@ func (p *PiGlow) Enable() error {
 	return p.conn.Write([]byte{0x00, 0x01})
 }
 
-// Setup enables normal operations, resets the internal registers, and enables
+// setup enables normal operations, resets the internal registers, and enables
 // all LED control registers
-func (p *PiGlow) Setup() error {
```
Comment (mattetti) [inline piglow/piglow.go]: if `Setup` stays exported (see #26) I would suggest to rename it. To me this is almost the real `Reset`. I don't know what the 3 registers do (doc would have been nice) but it sounds like `Setup` would be called the first time around or every time we want a clean slate, it does sound like a reset.

File: piglow/piglow.go [hunk 3]
```diff
@@ -83,19 +78,15 @@ func (p *PiGlow) Green(level int) error {
 	if err := p.conn.Write([]byte{0x04, byte(level)}); err != nil {
 		return err
 	}
-
 	if err := p.conn.Write([]byte{0x06, byte(level)}); err != nil {
```
Comment (mattetti) [inline piglow/piglow.go]: I am probably nitpicking but I would love to see these bytes being defined somewhere so their names can indicate what they are used for.

Comment (rakyll) [inline piglow/piglow.go]: Adding comments nearby what they are doing would be good to have. @zankich?

**Comment by rakyll** [comment]
/cc @zankich @mattetti 

Could you take a look please? I addressed all the comments from the previous PR.

File: piglow/piglow.go (hunk 3 above)
Comment (zankich) [inline piglow/piglow.go]: I can add constants for the registers which are being written to.

**Comment by mattetti** [comment]
LGTM

File: piglow/piglow.go (hunk 2 above)
Comment (zankich) [inline piglow/piglow.go]: How would you feel about removing this part from the PR while we discuss the proper way it should be called/named.

Comment (rakyll) [inline piglow/piglow.go]: Removed.

**Comment by rakyll** [comment]
Reverted the Setup related changes, PTAL.

**Comment by rakyll** [comment]
Merging this, please give feedback if there is any.

**Pull request merged**